packages/kit/src/runtime/server/page/render.js

```diff
@@ -72,7 +72,9 @@ export async function render_response({
 	/** @type {import('types').NormalizedLoadOutputCache | undefined} */
 	let cache;
 
-	if (error) {
+	const stack = error?.stack;
+
+	if (__SVELTEKIT_DEV__ && error) {
```

Comment:
I'm not sure we should include the `__SVELTEKIT_DEV__`. E.g. at my last company we would show you the stacktrace if you were logged in as an employee. We wouldn't show it to customers, but it was helpful for engineers to have access to it

Reply:
They do have access to it, in `handleError`. If we wanted to expose it more broadly than that then we can, but that's a much larger conversation than this change

Reply:
yeah, but it's harder to get at on a production machine vs having it just show up on the screen. I don't know that we should place restrictions on what people are allowed to do

Reply:
Sure — it's a conversation worth having, but it needs to be had separately. We currently prevent stack traces from appearing in production (at one point we didn't, but we were called out for bad security practices). This PR doesn't change that and I don't think this bugfix should get bogged down in that discussion, so let's figure it out elsewhere

Reply:
ah, I didn't realize we were doing that. this looked like new behavior